api/v1beta1/loadbalancerset_types.go

```diff
@@ -48,6 +48,9 @@ type LoadBalancerSetStatus struct {
 	// AvailableReplicas are the current running replicas.
 	// +optional
 	AvailableReplicas *int `json:"availableReplicas,omitempty"`
+	// Conditions contains condition information for a LoadBalancerSet.
+	// +optional
+	Conditions []metav1.Condition `json:"conditions,omitempty"`
```

Comment:
All other status objects are pointers, should we use here also a pointer?

Reply:
Let's keep the slice without a pointer. This is conformant with the API guidelines, ref https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#optional-vs-required